packages/dependencies/package.json

```diff
@@ -13,16 +13,23 @@
   "bugs": {
     "url": "https://github.com/solarwindscloud/solarwinds-apm-js/issues"
   },
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./dist/es/index.js",
+      "require": "./dist/cjs/index.js"
+    }
+  },
+  "main": "./dist/cjs/index.js",
   "files": [
-    "dist/"
+    "./src/",
```

Comment:
Makes debugging easier since the published sourcemaps will point to the files in `src`